packages/iris-grid/src/sidebar/TableCsvExporter.test.tsx

```diff
@@ -0,0 +1,57 @@
+// import React from 'react';
+// import { render, screen } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
+// import TableCsvExporter from './TableCsvExporter';
+// import IrisGridTestUtils from '../IrisGridTestUtils';
+
+// const COLUMN_NAMES = ['A', 'B', 'C', 'D'];
+
+// function makeTableCsvExporterWrapper({
+//   name = 'TEST',
+//   isDownloading = false,
+//   tableDownloadStatus = '',
+//   tableDownloadProgress = 0,
+//   tableDownloadEstimatedTime = undefined,
+//   onDownloadStart = () => null,
+//   onDownload = (
+//     fileName = null,
+//     frozenTable = null,
+//     tableSubscription = null,
+//     snapshotRanges = null
+//   ) => null,
+//   onCancel = () => null,
```

Comment:
In general in tests, for anywhere you need a default function, just pass in a `jest.fn()`. You can pass it in anywhere and you can inspect it to see if it's been called and with what values easily.